tests/run.bats

```diff
@@ -528,6 +528,11 @@ function configure_and_check_user() {
 	cid=$output
 	run_buildah 125 run --isolation=chroot --network=bogus $cid cat /etc/hosts
 	expect_output "checking network namespace: stat bogus: no such file or directory"
+
+	run_buildah run --isolation=chroot $cid cat /etc/hosts
```

Comment:
Why is this forcing isolation to chroot, even in cases where CI would be trying to use the default runtime?

Reply:
Just using existing tests.